package.json

```diff
@@ -7,7 +7,7 @@
     "================================================================================"
   ],
   "name": "@aurodesignsystem/auro-card",
-  "version": "2.0.4",
+  "version": "0.0.0",
```

Comment:
Update version

Reply:
This will be updated with the semantic-release commit back to the repo after the release. We do not manually edit this value.